Lib/test/test_asyncio/test_taskgroups.py

```diff
@@ -997,6 +997,31 @@ class MyKeyboardInterrupt(KeyboardInterrupt):
         self.assertIsNotNone(exc)
         self.assertListEqual(gc.get_referrers(exc), no_other_refs())
 
+    async def test_cancels_task_if_created_during_creation(self):
```

Comment:
this test requires https://github.com/python/cpython/pull/128553/files#diff-6f3f42fe6cdf1387de256a705e9f0d4c809ba04a2da04c317c9c79cfc4c5bc69R999 to be effective